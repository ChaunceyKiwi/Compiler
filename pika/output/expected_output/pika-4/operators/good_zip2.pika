[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]