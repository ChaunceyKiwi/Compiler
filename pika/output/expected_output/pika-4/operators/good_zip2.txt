[aaa, bbb, ccc, ddd, eee, fff]
[1, 4, 9, 16, 25, 36]
[[97, 1], [98, 4], [99, 9], [100, 16], [101, 25], [102, 36]]
